src/config.jl

```diff
@@ -102,27 +100,34 @@ function header_args(doc::WeaveDoc, out_path, mod, fig_ext, fig_path,
                             cache_path, cache, throw_errors,template,
                             highlight_theme, css,
                             pandoc_options, latex_cmd)
-    args = getvalue(doc.header, "options", Dict())
-    doctype = getvalue(args, "doctype", doc.doctype)
+    args = get(doc.header, "options", Dict())
+    doctype = get(args, "doctype", doc.doctype)
     args = combine_args(args, doctype)
-    informat = getvalue(args, "informat", :auto)
-    out_path = getvalue(args, "out_path", out_path)
+    informat = get(args, "informat", :auto)
+    out_path = get(args, "out_path", out_path)
     out_path == ":pwd" && (out_path = :pwd)
     isa(out_path, Symbol) || (out_path = joinpath(dirname(doc.source), out_path))
-    mod = Symbol(getvalue(args, "mod", mod))
-    fig_path = getvalue(args, "fig_path", fig_path)
-    fig_ext = getvalue(args, "fig_ext", fig_ext)
-    cache_path = getvalue(args, "cache_path", cache_path)
-    cache = Symbol(getvalue(args, "cache", cache))
-    throw_errors = getvalue(args, "throw_errors", throw_errors)
-    template = getvalue(args, "template", template)
+    if haskey(args, "mod")
+        m = args["mod"]
+        if m == "sandbox"
+            mod = :sandbox
+        else
+            @warn "The only valid header option for 'mod' is 'sandbox'. Ignoring value '$m'."
```

Comment:
It would be better to set `mod = Module(Symbol(m))` here. I don't think there is any reason not to be able to specify module name in header.

Reply:
Ah sorry, I misunderstood your previous comment. I assumed it should be restricted to ":sandbox" only.

In that case I think it's more reasonable to handle all `mod` options of type `Symbol` in the `run` method, such that `weave("test.jmd", mod = :sandbox3)` for a file `test.jmd` without header arguments and `weave("test_headers.jmd")` for a file `test_headers.jmd` with header arguments `mod: sandbox3` behave the same.